pages/index.js

```diff
@@ -12,10 +15,11 @@ export default function HomePage() {
         <ListHeader>
           <StyledListName>Shopping List</StyledListName>
           <StyledTotalItems>
-            {shoppingItems.length} items total{" "}
+            {shoppingItemsState.length} items total{" "}
```

Comment:
Once we have destructured the props (see previous comment). Then we need to make sure we use that prop name. 

```suggestion
            {shoppingItems.length} items total{" "}
```